nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -166,6 +166,19 @@ void SmacPlannerHybrid::configure(
   nav2_util::declare_parameter_if_not_declared(
     node, name + ".motion_model_for_search", rclcpp::ParameterValue(std::string("DUBIN")));
   node->get_parameter(name + ".motion_model_for_search", _motion_model_for_search);
+  std::string goal_heading_type;
+  nav2_util::declare_parameter_if_not_declared(
+    node, name + ".goal_heading_mode", rclcpp::ParameterValue("DEFAULT"));
+  node->get_parameter(name + ".goal_heading_mode", goal_heading_type);
+  GoalHeadingMode goal_heading_mode = fromStringToGH(goal_heading_type);
```

Comment:
```suggestion
  _goal_heading_mode = fromStringToGH(goal_heading_type);
```

Then you can remove the `else` and still check `_goal_heading_mode` for the UNKOWN condition. Its more concise

Reply:
done in https://github.com/ros-navigation/navigation2/pull/4127/commits/e06f1db001f89b867cbb5eb8116f53517e85df1e